frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -16,11 +17,11 @@ import { OCSStorageClassDropdown } from '../storage-class-dropdown';
 export const AddCapacityModal = withHandlePromise((props: AddCapacityModalProps) => {
   const { ocsConfig, close, cancel } = props;
   const dropdownUnits = {
-    Ti: 'Ti',
+    TiB: 'TiB',
```

Comment:
are these supposed to be changed for 4.2 ? Shouldnt 4.2 still use `Ti` ? (same for other occurrences)

Reply:
ack, I will create a separate commit for 4.2 specific changes as mentioned above